tests/shardddl1/run.sh

```diff
@@ -264,6 +264,12 @@ function DM_RECOVER_LOCK_CASE() {
 	# TrySync tb2: joined(a,c); tb1(a,c); tb2(a,b)
 	restart_master
 
+	# because we disabled auto-resume feature to satisfy other tests, we manually resume once to handle task was paused by
```

Comment:
Shall we check the task is stopped first?

Reply:
resume-task will also return `result: true` when task is not paused